URLSSLVerification added an argument (#14829)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_4_41.md

#### Scripts
##### URLSSLVerification

> - Added an argument that will allow the user to choose whether in case that the URL starts with HTTP and not HTTPS, to make the URL Suspicious.

Reviewer: ```suggestion
- Added the argument *set_http_as_suspicious* that will allow the user to choose whether to set the URL as Suspicious in case that the URL starts with HTTP and not HTTPS.
```
